reviews/review-01+2020-11-02.md

```diff
@@ -0,0 +1,305 @@
+way, and providing a motivation for each topic included.
+
+Other noteworthy features of the manuscript include:
+
+-   A commitment to defining terms, resulting in a thorough glossary
+
+-   Supplementary material to the main text in the form of callouts in
+    the chapters and several appendices
+
+-   Material that includes the human side of project development
+
+## What reservations do you have about this manuscript?
+
+I have no big-picture reservations about this manuscript. Some chapters
+would benefit from additional review or editing to further clarify their
+content; many of the exercises still need solutions, but I see these
+more as editorial concerns (addressed in the following questions) rather
+than significant structure or content reservations.
+
+## Content and structure
+
+In terms of content and structure, do you think the chapters need any
+revisions, such as additions or omissions? Please be as specific as
+possible.
+
+The opening chapter does an excellent job of setting up the audience and
+"syllabus" of the book. Its aim would be strengthened by adding a
+section on "How to Use this Book" or expanding the Syllabus (1.3) and/or
+Audience (1.2) sections with:
+
+-   Suggestions on how the book could be used. This could be as simple
+    as -- "Work through each chapter, following along with the commands
+    and changes described in the text. Try at least half of the
+    exercises for each chapter" -- to something as complex as an
+    appendix with suggestions for a suggested course plan, adapting
+    these materials to a lecture, finding a mentor or someone to help
+    you (if doing self-study), etc.
+
+-   Emphasizing the presence of exercises and solutions, as well as
+    recommendations for how to use them. I agree with the decision to
+    include the exercises at the end of each chapter, but I think a note
+    recommending that readers not wait until the end to check out some
+    of the available exercises would be a good idea.
+
+-   Outlining the standard chapter format: introduction, content,
+    exercises; identifying the callout box format as a mechanism to
+    supplement the main flow of text.
+
+My only additional comments are about rearranging/splitting specific
+material or chapters. None of these are strictly necessary -- I leave it
+up to the discretion of the authors as to whether they would improve the
+overall manuscript or not.
+
+-   The project organization description (1.4) seemed a bit out of place
+    in chapter 1 -- most of that chapter is about course **details**
+    (audience, installation, syllabus), but the Project Structure
+    section reads like course **content**. If the above suggestion is
+    added, this will even more be the case. I see a compelling argument
+    for making the current chapter 1 into a numberless prologue or
+    introduction and having chapter 1 be a very short chapter with the
+    Project Structure content.
```

Comment:
I like the idea of retaining section 1.4 (project organization) in the introduction chapter (which we might re-name "Getting Started" or something) and moving the rest of the current introduction content to a numberless prologue. 

We might want to also want to keep the reference to the software installation instructions in the "Getting Started" section rather than the prologue.